app/Services/PostService.php

```diff
@@ -73,6 +73,9 @@ public function update($signup, $data, $transactionId)
         // Save the new post in Customer.io, via Blink.
         if (config('features.blink') && $postOrSignup instanceof Post && $should_send_to_blink) {
             SendPostToBlink::dispatch($postOrSignup);
+
+            // Log that a post was created.
+            info('post_created', ['id' => $postOrSignup->id, 'signup_id' => $postOrSignup->signup_id]);
```

Comment:
Could this actually be a Post or a Signup? Asking because if it's a Signup I don't think it will have a `signup_id`

Reply:
I was gonna ask the same thing!! There's a very sneaky `instanceof` check above. 🕵️

Reply:
it's in a conditional that makes sure it's a `Post` instance. But ideally we wouldn't need to handle the Post or Signup situation.

Reply:
oooo I see the sneaky logic now! solid this looks all good then. YAY LOGGING!!!!!